code/FileAttachmentFieldCleanTask.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/**
+ * Delete all files being tracked that weren't saved against anything.
+ *
+ * WARNING: You must call 'FileAttachmentFieldTrack::untrack' against IDs on custom-built forms or you
+ *          -will- remove files accidentally.
+ *
+ * @package  unclecheese/silverstripe-dropzone
+ */
+class FileAttachmentFieldCleanTask extends BuildTask {
+    protected $title = "File Attachment Field - Clear all tracked files that are older than 1 hour";
+    
+    protected $description = 'Delete files uploaded via FileAttachmentField that aren\'t attached to anything.';
+    
+    public function run($request) {
+        $files = FileAttachmentFieldTrack::get()->filter(array('Created:LessThanOrEqual' => date('Y-m-d H:i:s', time()-3600)));
+        $files = $files->toArray();
+        if ($files) {
```

Comment:
Not sure this needs to be converted to an array, as the DataList is already iterable. Use `if($files->exists())` as your guard.